vendor/github.com/coreos/etcd/scripts/build-docker

```diff
@@ -40,4 +40,4 @@ cp ${BINARYDIR}/etcd ${BINARYDIR}/etcdctl ${IMAGEDIR}
 
 cat ./${DOCKERFILE} > ${IMAGEDIR}/Dockerfile
 
-docker build -t ${TAG}:${VERSION} ${IMAGEDIR}
+docker build --force-rm=true -t ${TAG}:${VERSION} ${IMAGEDIR}
```

Comment:
Please remove changes from the vendor directory. There are 3rd party dependencies and all changes will be replaces when dependencies are updated.